web-dashboard.md

```diff
@@ -0,0 +1,79 @@
+The Anbox Cloud Dashboard offers a web GUI from where developers can create, manage, and even stream applications from their web browser.
+The dashboard is useful if you are new to Anbox Cloud or want a simple management interface.
+
+Starting in 1.9.0, the dashboard replaces the old Dev UI.
+
+> **Note**: The web dashboard is more oriented toward developers than end-users. However, it only uses available APIs and SDKs, and can be a good example to build your own web-based platform.
+
+![Screenshot 2021-02-10 at 14.33.52|690x322](upload://azCr6HYSx9mJZ82K2CPdTb3IS34.png) 
+
+# Installation
+
+The dashboard comes pre-installed when you deploy [Anbox Cloud with the streaming stack](https://discourse.ubuntu.com/t/install-anbox-cloud/17744) or the [Anbox Cloud Appliance](https://discourse.ubuntu.com/t/install-appliance/22681). It sits behind a reverse proxy for performance and security reasons.
+
+# Usage
+
+To access the dashboard, go to `https://<your-machine-address>/`.
+
+> **Hint**: The dashboard uses self-signed certificates. You might see a warning from your browser and have to accept the certificates manually.
+
+## Granting access
+
+Instead of implementing its own user management, the dashboard relies on OAuth for user authentication. The only OAuth provider supported right now is [Ubuntu One](https://login.ubuntu.com/).  Future versions of Anbox Cloud will allow using other OAuth providers.
```

Comment:
```suggestion
Instead of implementing its own user management, the dashboard relies on OAuth for user authentication. The only OAuth provider supported right now is [Ubuntu One](https://login.ubuntu.com/). Future versions of Anbox Cloud will allow using other OAuth providers.
```